bossggle.js

```diff
@@ -0,0 +1,28 @@
+// setup letters
+
+var playWord = "";
+
+setupLetters();
+
+setupListeners();
+
+$(document).ready(function() {
+	$("#submit").on("click", function() {
+		givePoints();
+		addWord();
+		playWord = ""
```

Comment:
This is necessary. In between the beginning and when a player clicks submit, playWord has changed to the word that they're submitting.